flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/serializer/RowDataToProtoSerializer.java

```diff
@@ -202,17 +217,16 @@ public Object toProtoValue(
                 | IllegalStateException
                 | IndexOutOfBoundsException
                 | DateTimeException e) {
-            LOG.info(
+            LOG.error(
                     String.format(
                             "Expected Type: %s at Field Number %d for Logical Type: %s.%nError: %s",
                             fieldDescriptor.getType().name(),
                             fieldNumber,
                             fieldType.getTypeRoot().name(),
                             e));
             throw new IllegalArgumentException(
-                    String.format("Error while converting value. %nErrors: %s to proto value", e));
+                    String.format("Error while converting value. %nErrors: %s", e));
```

Comment:
Fixed.

Reply:
Lets keep track of the terminology we use in error logs and exceptions, so that we maintain consistency when addressing the same entities across different error scenarios.
For example: referring to BQ table's row proto as `BQ proto equivalent`. You can choose any phrase that makes sense to you, but pls use them consistently.